engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -259,15 +259,18 @@ public void processAggregationSync(Event event) {
     private void processAggregateEmailsByAggregationKey(AggregationCommand aggregationCommand, Event aggregatorEvent) {
 
         Log.infof("Processing received aggregation command: %s", aggregationCommand);
-        Optional<Application> app = applicationRepository.getApplication(aggregationCommand.getAggregationKey().getBundle(), aggregationCommand.getAggregationKey().getApplication());
-        if (app.isPresent()) {
-            String eventTypeDisplayName = String.format("%s - %s - %s",
-                aggregatorEvent.getEventTypeDisplayName(),
-                app.get().getDisplayName(),
-                app.get().getBundle().getDisplayName()
-            );
-            eventRepository.updateEventDisplayName(aggregatorEvent.getId(), eventTypeDisplayName);
-        }
+        Application app = applicationRepository.getApplication(aggregationCommand.getAggregationKey().getBundle(), aggregationCommand.getAggregationKey().getApplication())
+                .orElseThrow(() -> {
+                    String exceptionMsg = String.format("Application not found: %s/%s", aggregationCommand.getAggregationKey().getBundle(), aggregationCommand.getAggregationKey().getApplication());
```

Comment:
```suggestion
                    String exceptionMsg = String.format("Application not found: %s - /%s", aggregationCommand.getAggregationKey().getBundle(), aggregationCommand.getAggregationKey().getApplication());
```

Reply:
There's already a separator (`/`) in the existing message. This should never be logged anyway so I'm merging without changing this.